src/main/java/uk/gov/companieshouse/model/validator/CompanyNumberValidator.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.companieshouse.model.validator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+import org.springframework.stereotype.Component;
+
+import uk.gov.companieshouse.config.CompanyNumberConfiguration;
+
+@Component
+public class CompanyNumberValidator implements ConstraintValidator<ValidCompanyNumber, String> {
+
+    private final Pattern prefixListRegex = Pattern.compile("(?i)^([A-Z][A-Z]?)\\b(,[A-Z][A-Z]?)*$");
```

Comment:
can we made this static?

Reply:
What's the benefit of that?

Reply:
this being a static constant allows you to access it without creating an instance of the class and it's common to all objects of the class

Reply:
just noticed this is only used in one place... we could hide only the regex behind a constant